console/src/main/scala/io/joern/console/cpgcreation/ImportCode.scala

```diff
@@ -150,6 +193,17 @@ class ImportCode[T <: Project](console: io.joern.console.Console[T])(implicit
     }
   }
 
+  class JsFrontend(name: String, language: String, description: String, extension: String)
+      extends SourceBasedFrontend(name, language, description, extension) {
+    override def apply(inputPath: String, projectName: String, args: List[String]): Cpg = {
+      if (!File(inputPath).isDirectory) {
```

Comment:
please add a comment (and may error message?) that JS doesn't support this, just to clarify